metadata.json

```diff
@@ -1,12 +1,17 @@
 {
   "name": "puppet-extlib",
-  "version": "0.11.1-rc0",
-  "source": "https://github.com/voxpupuli/puppetcommunity-extlib",
-  "author": "puppetcommunity",
+  "version": "0.11.1-rc1",
```

Comment:
I believe no metadata should be updated in this PR. Please submit this in a separate PR.